tests/dags/common/test_urls.py

```diff
@@ -96,6 +96,12 @@ def test_validate_url_string_caches_tls_support(clear_tls_cache, monkeypatch):
     mock_get.assert_called_once()
 
 
+def test_validate_url_string_keeps_trailing_slash():
+    url_string = "https://wordpress.org/photos/photo/5262839486/"
+    actual_validated_url = urls.validate_url_string(url_string, strip_slash=False)
```

Comment:
@thedevhaider , can we add another test case  when `strip_slash=True`? This will confirm `strip_slash` is working for both the cases.

Reply:
@amustaque97 Yeah this makes sense. Thanks for pointing this out!